nifi-toolkit/nifi-toolkit-admin/src/main/groovy/org/apache/nifi/toolkit/admin/client/NiFiClientFactory.groovy

```diff
@@ -17,38 +17,19 @@
 package org.apache.nifi.toolkit.admin.client
 
 import org.apache.commons.lang3.StringUtils
-import org.apache.nifi.security.util.CertificateUtils
+import org.apache.http.conn.ssl.DefaultHostnameVerifier
 import org.apache.nifi.util.NiFiProperties
-import org.slf4j.Logger
-import org.slf4j.LoggerFactory
-import org.apache.http.conn.ssl.DefaultHostnameVerifier;
 
-
-
-import javax.naming.ldap.LdapName
-import javax.naming.ldap.Rdn
-import javax.net.ssl.HostnameVerifier
 import javax.net.ssl.KeyManagerFactory
 import javax.net.ssl.SSLContext
-import javax.net.ssl.SSLPeerUnverifiedException
-import javax.net.ssl.SSLSession
 import javax.net.ssl.TrustManagerFactory
 import javax.ws.rs.client.Client
 import javax.ws.rs.client.ClientBuilder
-import java.security.KeyManagementException
-import java.security.KeyStore
-import java.security.KeyStoreException
-import java.security.NoSuchAlgorithmException
-import java.security.SecureRandom
-import java.security.UnrecoverableKeyException
-import java.security.cert.Certificate
+import java.security.*
```

Comment:
We can't import wildcards. This might be your IDE doing it automatically (in which case, loading the `checkstyle.xml` will prevent this in the future).

Reply:
Woopsie